db/test/versions/0015_test.js

```diff
@@ -0,0 +1,172 @@
+const _ = require('lodash');
+const helper = require('../helper');
+const testing = require('taskcluster-lib-testing');
+const assert = require('assert').strict;
+const Entity = require('taskcluster-lib-entities');
+
+const THIS_VERSION = 15;
+const PREV_VERSION = THIS_VERSION - 1;
+
+// (adapted from services/github/src/data.js) NOTE: this will be
+// removed when tc-lib-entities is dropped from the repository
+const Builds = Entity.configure({
+  version: 1,
+  partitionKey: Entity.keys.StringKey('taskGroupId'),
+  rowKey: Entity.keys.ConstantKey('taskGroupId'),
+  properties: {
+    organization: Entity.types.String,
+    repository: Entity.types.String,
+    sha: Entity.types.String,
+    taskGroupId: Entity.types.String,
+    state: Entity.types.String,
+    created: Entity.types.Date,
+    updated: Entity.types.Date,
+  },
+}).configure({
+  version: 2,
+  properties: {
+    organization: Entity.types.String,
+    repository: Entity.types.String,
+    sha: Entity.types.String,
+    taskGroupId: Entity.types.String,
+    state: Entity.types.String,
+    created: Entity.types.Date,
+    updated: Entity.types.Date,
+    // GitHub installation ID that comes from the webhook
+    // Needed for authentication in statusHandler
+    installationId: Entity.types.Number,
+  },
+  migrate: function(item) {
+    item.installationId = 0;
+    return item;
+  },
+}).configure({
+  version: 3,
+  properties: {
+    organization: Entity.types.String,
+    repository: Entity.types.String,
+    sha: Entity.types.String,
+    taskGroupId: Entity.types.String,
+    state: Entity.types.String,
+    created: Entity.types.Date,
+    updated: Entity.types.Date,
+    installationId: Entity.types.Number,
+    eventType: Entity.types.String,
+  },
+  migrate: function(item) {
+    item.eventType = 'Unknown Event';
+    return item;
+  },
+}).configure({
+  version: 4,
+  properties: {
+    organization: Entity.types.String,
+    repository: Entity.types.String,
+    sha: Entity.types.String,
+    taskGroupId: Entity.types.String,
+    state: Entity.types.String,
+    created: Entity.types.Date,
+    updated: Entity.types.Date,
+    installationId: Entity.types.Number,
+    eventType: Entity.types.String,
+    eventId: Entity.types.String,
+  },
+  migrate: function(item) {
+    item.Id = 'Unknown';
+    return item;
+  },
+}).configure({
+  version: 5,
+  properties: {
+    organization: Entity.types.String,
+    repository: Entity.types.String,
+    sha: Entity.types.String,
+    taskGroupId: Entity.types.String,
+    state: Entity.types.String,
+    created: Entity.types.Date,
+    updated: Entity.types.Date,
+    installationId: Entity.types.Number,
+    eventType: Entity.types.String,
+    eventId: Entity.types.String,
+  },
+  migrate: function(item) {
+    // Delete Id because it was mistakenly set in the last migration
+    delete item.Id; // Just in case the migrations are chained
+    item.eventId = item.eventId || 'Unknown';
+    return item;
+  },
+});
```

Comment:
You can shorten this to a single configure call with the final set of properties.  The `version` and `migrate` fields are ignored anyway.